_studio/mfx_lib/cmrt_cross_platform/include/cmvm.h

```diff
@@ -984,7 +983,6 @@ matrix<T, R, C>::matrix(const matrix_ref<T2, R2, C2>& src, const uint sat)
     : array_2d_(data)
 #endif
 {
-    static const bool conformable = check_true<R*C == R2*C2>::value;
     assert(R*C == R2*C2);
```

Comment:
And of the above was intended to be a STATIC_ERROR, then question is why we have same assert here? Answer to this is simple, CM_STATIC_ERROR is defined to nothing:)! I suggest to define it properly and remove assert() calls altogether.